raiden/network/proxies/user_deposit.py

```diff
@@ -317,7 +319,7 @@ def deposit(
         current_inflight = self._inflight_deposits.get(beneficiary)
         if current_inflight is not None and current_inflight.total_deposit >= total_deposit:
             current_inflight.async_result.get()
-            return
+            return None
```

Comment:
Should we return the txhash of the inflight tx, here?

Reply:
Yes, we should! Implemented in 772f74d2cfefcb0d6f5b3d758a7f4cb1c1270c0b